src/components/content/order/orderStatus/OrderSubmitResultDetails.tsx

```diff
@@ -3,10 +3,43 @@
  * SPDX-FileCopyrightText: Huawei Inc.
  */
 
+import { Typography } from 'antd';
+import { CheckOutlined, CopyOutlined } from '@ant-design/icons';
+import React from 'react';
+import { myServicesRoute } from '../../../utils/constants';
+import { createSearchParams, useNavigate } from 'react-router-dom';
+
 function OrderSubmitResultDetails({ msg, uuid }: { msg: string | JSX.Element; uuid: string }): JSX.Element {
+    const { Paragraph } = Typography;
+    const navigate = useNavigate();
     return (
         <>
-            Request ID: <b>{uuid}</b>
+            Service ID:{' '}
+            <div
+                onClick={() => {
```

Comment:
when the mouse hover happens, it must also be visible that this is a link. 
It can be achieved by CSS property - `cursor: pointer;`